### ISSUE TITLE
Add an "examples" directory

### ISSUE THREAD

**Issue by prestonvasquez** [alpstable/gidari#371]
The project needs an "examples" directory and the first example should be for the "[csvpb](https://github.com/alpstable/csvpb)" package. The directory should have the following layout:

```
examples/
├─ csvpb/
│  ├─ go.mod
│  ├─ go.sum
│  ├─ main.go
```

where `main.go` is an illustration for how to use gidari and csvpb to write HTTP response data to io.

**Comment by prestonvasquez** [comment]
@Conor-Fleming If you're still interested in the project, this would be a good ticket to get introduced to the updated architecture. We want to combine the [csvpb](https://github.com/alpstable/csvpb/blob/main/csvpb_example_test.go#L19) example with [this](https://github.com/alpstable/gidari/blob/main/gidari_example_test.go#L91) Gidari example. That is, write the ASOIAF data to stdout as CSV.

**Comment by Conor-Fleming** [comment]
Yeah that would be great. I can take this one. Thanks! @prestonvasquez

**Comment by prestonvasquez** [comment]
@Conor-Fleming Make sure you take the latest version of main, I just had to fix a small bug.

**Comment by Conor-Fleming** [comment]
@prestonvasquez Based on the layout above you want to add seperate modules within each directory of the examples directory. I ran into some issues when trying to set it up that way. Based on my (limited) understanding we'd want to use `go.work` files?? but I would probably need some guidance if you wanted to set it up that way. For now I left it as a stand alone `main.go` within `examples/csvpb`. 

Beyond that I also made the request endpoints a little bit more specific just to improve the readability of the output. I can change this if you want. Ill open a PR shortly. Thanks!
